process.py

```diff
@@ -385,12 +386,22 @@ def get_provider_code(directory_name=None):
 
 
 def main():
+    dirname = "static/data/lms"
     parser = argparse.ArgumentParser()
-    parser.add_argument('--path', type=str, default="static/data/lms")
-    parser.add_argument('--batch_size', type=int, default=100)
+    parser.add_argument('--specific_folder', dest='specific_folder', action='store_true', default=False)
     parser.add_argument('--delete_all', dest='delete_all', action='store_true', default=True)
+    parser.add_argument('--path', type=str, default=dirname)
+    parser.add_argument('--batch_size', type=int, default=100)
     parser.add_argument('--provider_code', type=int)
     args = parser.parse_args()
+
+    if args.specific_folder:
+        dirname = tkFileDialog.askdirectory(initialdir=os.path.abspath(dirname),  title='Please select a directory')
+        if args.delete_all:
+            is_delete_reuqired = raw_input("Are you sure you want to delete all the current data? type y to delete, type n otherwise\n")
```

Comment:
`is_delete_reuqired` should be `is_delete_required`. Or maybe just `confirm_delete_all`.